app/models/bpldc/basic_genre.rb

```diff
@@ -2,6 +2,14 @@
 
 module Bpldc
   class BasicGenre < Bpldc::Nomenclature
-    belongs_to :authority, inverse_of: :basic_genres, class_name: 'Bpldc::Authority'
+    belongs_to :authority, inverse_of: :basic_genres, class_name: 'Bpldc::Authority', touch: true
```

Comment:
Do we need `touch: true`? If we update a genre term, I don't think we need to update the parent authority? (Applies to all `Nomenclature` subclasses.)

Reply:
Its in order to clear the cache for authorities. SInce touching triggers the after commit callback